clean-abap/CleanABAP.md

```diff
@@ -2181,6 +2200,29 @@ which will never have an alternative implementation and will never be mocked in
 > [Interfaces vs. abstract classes](sub-sections/InterfacesVsAbstractClasses.md)
 describes why this also applies to classes that overwrite inherited methods.
 
+#### Use alias for using interface members
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Methods](#methods) > [Methods: Object orientation](#methods-object-orientation) > [This section](#use-alias)
+
+While calling interface methods or using interface attributes, using alias is much cleaner.
+
+```ABAP
+class clean_class definition public inheriting from if_clean_interface.
```

Comment:
syntax error? `inheriting from if_clean_interface`

Reply:
I did not check the code in editor and i did not close the class definition. Is that why?

Reply:
A class can not inherit from an interface in abap ;)

Reply:
My bad. I will correct the code. Thanks for pointing out.